contracts/modules/validators/K1Validator.sol

```diff
@@ -252,22 +245,9 @@ contract K1Validator is IValidator, ERC7739Validator {
         }
 
         // verify signer
-        try this.recoverSigner(hash, signature) returns (address recoveredSigner) {
-            if (recoveredSigner == owner) {
-                return true;
-            }
-        } catch {
-            // If recovery fails, we'll continue to the next check
-        }
-
-        try this.recoverSignerFromEthSignedMessage(hash, signature) returns (address recoveredSigner) {
-            if (recoveredSigner == owner) {
-                return true;
-            }
-        } catch {
-            // If recovery fails, we'll return false
-        }
-
+        // owner can not be zero address in this contract
```

Comment:
can't the owner be zero here: https://github.com/bcnmy/nexus/pull/198/files#diff-0de611a725083823f818063145fae9df78b92f67f62ed9528d5337dd17aa7e10R74-R76 ?

Reply:
oh, I thought we had zero address check in place. I think we can enforce this in onInstall and create informational issue. @0xbok

Reply:
https://github.com/bcnmy/nexus/pull/199